### PR TITLE
Improve CI matrix flags

### PR DESCRIPTION
**Pull request by Meakk**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -26,15 +26,15 @@ jobs:
...
         include:
-          - enable_raytracing: true
+          - enable_raytracing: raytracing
           - vtk_version: v9.0.0
-            enable_raytracing: false
+            enable_raytracing: no-raytracing
           - os: macos-latest
-            enable_raytracing: false #598
+            enable_raytracing: no-raytracing #598
           - macos_bundle: false
```
Comment (mwestphal) [inline .github/workflows/ci.yml:34]: do the same with macos_bundle ?

Comment (Meakk) [reply .github/workflows/ci.yml:34]: yes I will

**Comment by mwestphal** [comment]
Doesn't look like what I'd have expected:

a


We may have been testing wrong for a while

**Comment by mwestphal** [comment]
superseeded by #783

**Pull request closed without merging**